test/selenium_tests/framework.py

```diff
@@ -145,6 +145,10 @@ def func_wrapper(self, *args, **kwds):
                 self.reset_driver_and_session()
             try:
                 return f(self, *args, **kwds)
+            except unittest.SkipTest:
+                dump_test_information(self, f.__name__)
+                # Don't retry if we have purposely decided to skip the test.
+                raise
             except Exception:
                 dump_test_information(self, f.__name__)
                 if retry_attempts < GALAXY_TEST_SELENIUM_RETRIES:
```

Comment:
It was a bit hard for me to find in the console log that a test failed and was retried (e.g. both JupyterTestCase tests in https://jenkins.galaxyproject.org/job/docker-selenium/1716/console), the only clue for me was that `Binding default provider to: org.openqa.selenium.chrome.ChromeDriverService` was repeated. Can we insert a log statement after this line?